app/code/Magento/Catalog/Test/Unit/Model/ResourceModel/Product/GalleryTest.php

```diff
@@ -476,4 +476,20 @@ public function testCountImageUses()
             ->willReturn($results);
         $this->assertEquals($this->resource->countImageUses(1), count($results));
     }
+
+    public function testRemoveImagesPositionExcludeDefaultStore()
```

Comment:
To test interaction with a database there should be integration test instead.

Reply:
@orlangur can you specify for @XxXgeoXxX what you'd expect, or possibly even help implement it?

Reply:
@orlangur I'll accept it for know, as it's the same as the accepted PR for 2.3 branch. If you can specify the needed test, please do to, and @XxXgeoXxX can add it as a new PR.